Packs/CVE_2025_31324/Playbooks/playbook-CVE-2025-31324_-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2835 @@
+
+        **Note**: Running this script is a manual action. You are responsible for any outcomes or issues that may arise from its execution.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "35"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3996.75,
+          "y": 4073
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: 247e1217-8c75-48c1-836c-ec783ba881b0
+    type: regular
+    task:
+      id: 247e1217-8c75-48c1-836c-ec783ba881b0
+      version: -1
+      name: Close investigation
+      description: Close the current alert
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "35"
+    scriptarguments:
+      closeReason:
+        simple: No further actions are required.
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4437.5,
+          "y": 4073
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "38":
+    id: "38"
+    taskid: 3cbbc09a-3ae3-4a75-8be8-b36cf8513027
+    type: title
+    task:
+      id: 3cbbc09a-3ae3-4a75-8be8-b36cf8513027
+      version: -1
+      name: Resolution
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "34"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 4217,
+          "y": 3813
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "72":
+    id: "72"
+    taskid: 2678fec9-a99a-4d7f-ab2b-7481aa92ba84
+    type: condition
+    task:
+      id: 2678fec9-a99a-4d7f-ab2b-7481aa92ba84
+      version: -1
+      name: Are there any compromised hosts?
+      description: Are there any compromised hosts?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "28"
+      "yes":
+      - "73"
+      - "81"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: incident.agentsid
+            iscontext: true
+          right:
+            value: {}
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: incident.agentid
```

Comment:
the path to agentid should be with foundIncidnet cause only there you will see the Alert's `agentid`